sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -257,3 +255,59 @@ def get_topic_sender(self, topic_name, **kwargs):
         )
 
         return sender
+
+    def get_subscription_receiver(self, topic_name, subscription_name, **kwargs):
+        # type: (str, str, Any) -> ServiceBusReceiver
+        """Get ServiceBusReceiver for the specific subscription under the topic.
+
+        :param str topic_name: The name of specific Service Bus Topic the client connects to.
+        :param str subscription_name: The name of specific Service Bus Subscription
+         under the given Service Bus Topic.
+        :keyword mode: The mode with which messages will be retrieved from the entity. The two options
+         are PeekLock and ReceiveAndDelete. Messages received with PeekLock must be settled within a given
+         lock period before they will be removed from the queue. Messages received with ReceiveAndDelete
```

Comment:
queue->subscription (here and elsewhere in the docs)

Reply:
updated